build.sbt

```diff
@@ -46,6 +46,7 @@ ThisBuild / developers := List(
 
 ThisBuild / crossScalaVersions := Seq(scala212, scala213, scala3)
 ThisBuild / scalaVersion := scala213
+ThisBuild / tlJdkRelease := Some(11)
```

Comment:
@ybasket are you ok with this?

Reply:
I'm slightly unsure on whether we lose more by losing JDK 8 or Play! JSON 🙈 Ideally, we could bump the Java version only for Play!.

But Java 8 is really old and we're not targeting big legacy enterprise here, so let's go for it.

Reply:
TBH Recently I even wanted to bump to 17...